src/Umbraco.Web/Properties/AssemblyInfo.cs

```diff
@@ -12,7 +12,7 @@
 
 // Umbraco Cms
 [assembly: InternalsVisibleTo("Umbraco.Web.UI")]
-
+[assembly: InternalsVisibleTo("Umbraco.ModelsBuilder.Embedded")]
```

Comment:
why is this needed?

Reply:
We use `UmbracoFeatures.Disabled` to removed the embedded modelsbuilder dashboard if we detect the full version.

Reply:
never mind, found it. this should remove the dashboard and the controller:

    // remove the embedded dashboard
    composition.Dashboards().Remove<ModelsBuilderDashboard>();

    // remove the embedded controller
    composition.WithCollectionBuilder<UmbracoApiControllerTypeCollectionBuilder>()
        .Remove<ModelsBuilderDashboardController>();